src/status_im2/integration_test/wallet.cljs

```diff
@@ -0,0 +1,54 @@
+(ns status-im2.integration-test.wallet
+  (:require
+    [cljs.test :refer [deftest is]]
+    [clojure.string :as string]
+    [day8.re-frame.test :as rf-test]
+    [re-frame.core :as rf]
+    [status-im.events]
+    [status-im.multiaccounts.logout.core :as logout]
+    [status-im.subs.root]
+    [status-im.transport.core :as transport]
+    [status-im2.events]
+    [status-im2.integration-test.helpers :as helpers]
+    [status-im2.navigation.core]
+    [status-im2.subs.root]
+    [taoensso.timbre :as log])
+  (:require-macros
+    [status-im2.integration-test.macros :refer [with-app-initialized with-account]]))
+
+(deftest create-wallet-account-test
+  (log/info "====== create-wallet-account-test ==================")
+  (rf-test/run-test-async
+    (with-app-initialized
+      (with-account
+        (helpers/assert-messenger-started)
+        (helpers/create-new-account!)                       ; create a new account
+        (rf-test/wait-for
+          [:wallet-legacy.accounts/account-stored]
+          (helpers/assert-new-account-created)              ; assert account was created
+          (helpers/logout!)
+          (rf-test/wait-for [::logout/logout-method]))))))
+
+(deftest back-up-seed-phrase-test
+  (log/info "========= back-up-seed-phrase-test ==================")
+  (rf-test/run-test-async
+    (with-app-initialized
+      (with-account
+        (helpers/assert-messenger-started)
```

Comment:
Every time `with-account` is used in the tests, `assert-messenger-started` is called as well. I think it would be better to move this check to the macro itself because it's always desirable.